worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/Clipboard.java

```diff
@@ -68,4 +71,19 @@
     default boolean hasBiomes() {
         return false;
     }
+
+    /**
+     * Returns a clipboard with a given transform baked in.
+     *
+     * <p>
+     * Note: If an identity transform is given, the original clipboard will be returned unedited.
```

Comment:
I'd like to be less specific about the exact implementation, and just say this instead:
```suggestion
     * Note: This method may return the same clipboard object, if a copy is needed then you should check the returned value for identity equality and copy if needed.
```